src/main/kotlin/com/openlattice/search/renderers/CAREIssueAlertEmailRenderer.kt

```diff
@@ -0,0 +1,187 @@
+package com.openlattice.search.renderers
+
+import com.google.common.collect.Maps
+import com.openlattice.data.requests.NeighborEntityDetails
+import com.openlattice.edm.EdmConstants
+import com.openlattice.mail.RenderableEmailRequest
+import com.openlattice.search.requests.PersistentSearch
+import org.apache.olingo.commons.api.edm.FullQualifiedName
+import java.time.OffsetDateTime
+import java.util.*
+
+
+private const val FROM_EMAIL = "courier@openlattice.com"
+private const val TEMPLATE_PATH = "mail/templates/shared/CAREIssueAlertTemplate.mustache"
+
+/* FQNs */
+private val PERSON_ID_FQN = FullQualifiedName("nc.SubjectIdentification")
+private val FIRST_NAME_FQN = FullQualifiedName("nc.PersonGivenName")
+private val MIDDLE_NAME_FQN = FullQualifiedName("nc.PersonMiddleName")
+private val LAST_NAME_FQN = FullQualifiedName("nc.PersonSurName")
+private val STATUS_FQN = FullQualifiedName("ol.status")
+private val DESCRIPTION_FQN = FullQualifiedName("ol.description")
+private val PRIORITY_FQN = FullQualifiedName("ol.priority")
+private val CATEGORY_FQN = FullQualifiedName("ol.category")
+private val TITLE_FQN = FullQualifiedName("ol.title")
+private val COMPLETED_DATE_TIME_FQN = FullQualifiedName("date.completeddatetime")
+private val ENTRY_UPDATED_FQN = FullQualifiedName("general.entryupdated")
+
+
+/* METADATA TAGS */
+private const val PERSON_ENTITY_SET_ID_METADATA = "personEntitySetId"
+private const val STAFF_ENTITY_SET_ID_METADATA = "staffEntitySetId"
+private const val TIME_ZONE_METADATA = "timezone"
+private const val ASSIGNED_TO_ENTITY_SET_ID_METADATA = "assignedToEntitySetId"
+private const val REPORTED_ENTITY_SET_ID_METADATA = "reportedEntitySetId"
+
+/*
+*
+* Search metadata is expected to contain the following fields:
+* {
+*   "personEntitySetId": <UUID>,
+*   "staffEntitySetId": <UUID>,
+*   "assignedToEntitySetID": <UUID>,
+*   "reportedEntitySetID": <UUID>,
```

Comment:
looks like these don't quite match the metadata tags above -- `reportedEntitySetID` vs `reportedEntitySetId`, and `assignedToEntitySetID` vs `assignedToEntitySetId`. Also looks like `alertName` is not needed.